README.md

```diff
@@ -31,3 +31,42 @@ state = "ReadyForProof"
 +++
 ```
 The rest of the file can include e.g. an informal description of the proof or any useful hints.
+
+
+## Definition of Key-Value pairs
+
+**Type:** Theorem, Definition, Axiom
+
+**Reference**: type: string, Usually a book, a url, with page and/or theorem number.
+
+**Dependencies:** JS-Array of strings, they have to exist somewhere in the database.
```

Comment:
Actually dependencies don't have all to exist in the database, the whole point is to devise a path that is not fully proven yet.

Reply:
Actually it was really interesting, once I saw your post on the mailing list I immediately started my research.
It was quite cool how Terence Tao made the blueprint. It looked really nice that you can structure theorems so that many people can work collaboratively.

I hope we can do similar things as my fundamantal knowledge regarding Galois fields has some big gaps. I'm more in the camp of using existing theorems I know are true rather than developing fundamental theory, as the second one is harder and should be reviewed with more scrutiny.

I will fix the documentation once I find time, either later today or tomorrow.